File: python/ejemplo_09.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-

import gi
try:
    gi.require_version('Gtk', '3.0')
except ValueError:
    exit(1)
from gi.repository import Gtk


if __name__ == '__main__':
    dialog = Gtk.FontChooserDialog()
    try:
        if dialog.run() == Gtk.ResponseType.OK:
            print(dialog.get_font())
            exit(0)
    except KeyboardInterrupt:
        pass
    exit(1)

File: javascript/ejemplo_09.js
#!/usr/bin/gjs

imports.gi.versions.Gtk = '3.0';
const {Gtk} = imports.gi;

Gtk.init(null);

function show_dialog(){
    let dialog = new Gtk.FontChooserDialog({
        title: 'https://www.atareao.es'
    });
    try{
        if(dialog.run() == Gtk.ResponseType.OK){
            print(dialog.get_font());
            return 0;
        }
    }catch(error){
    }
    return 1;
}
show_dialog();
